webapp/public_rest_api/parking_sites/parking_sites_rest_api.py

```diff
@@ -82,6 +82,8 @@ class ParkingSiteListMethodView(ParkingSiteBaseMethodView):
             Parameter('lat', schema=NumericField(), example=55.5),
             Parameter('lon', schema=NumericField(), example=55.5),
             Parameter('radius', schema=NumericField(), description='Radius, in m', example='3500'),
+            Parameter('limit', schema=IntegerField(), description='Limit results'),
+            Parameter('start', schema=IntegerField(), description='Start of search query.'),
```

Comment:
A more common name would be `offset` instead of `start`.

Reply:
Added some explanation to pagination.